app/assets/src/components/views/SampleViewV2/ReportTable.jsx

```diff
@@ -229,7 +231,22 @@ class ReportTableV2 extends React.Component {
             )}
           </div>
           <div className={cx(rowData.taxLevel == "species" && cs.speciesName)}>
-            {cellData}
+            <span
+              className={cx(cs.taxonName, cellData || cs.missingName)}
```

Comment:
Names display properly when the report is first loaded, but the styling for the names seems to be flipped after the type is toggled. The names will be grayed out when set to `Scientific` instead of `Common`.

**When the page is refreshed:**
![image](https://user-images.githubusercontent.com/53838890/69390317-ae230080-0c83-11ea-959d-73a73e92834b.png)

**After toggling the name type:**
![image](https://user-images.githubusercontent.com/53838890/69390204-48cf0f80-0c83-11ea-8bb8-0ae9d22f4bd5.png)

![image](https://user-images.githubusercontent.com/53838890/69390149-2b01aa80-0c83-11ea-9e1a-e5cea0883a9a.png)

Reply:
Good catch.
Fixed.